### PR TITLE
Add bundle

### PR DESCRIPTION
**Pull request by WRFitch**
## Proposal
Adds k8s bundle, CI/CD pipelines, and integration tests. 

## Context 
- The way these pipelines work is as follows:
  1. `on_bundle_update_available.yaml` runs a daily cron job to check for updated bundle information in charmhub. When there's a new pgbouncer or postgresql release, it creates a new postgresql-k8s-bundle.yaml and opens a PR. 
  2. CI runs parallel tests on the PR, as normal. 
  3. On release, release.yaml runs the same tests as before, and releases to charmhub. 
- Files you don't need to review thoroughly: 
  - LICENSE.md
  - All of the test files themselves are copied wholesale from the pgbouncer-k8s charm, with edits to ensure they use the bundle. 
  - This includes pgb.py
- The certificate operator is going to be added in a future PR. Just doing the basics for now. 
- bundle.yaml doesn't contain storage or resource information. There's currently [a bug where bundles don't like these fields](https://bugs.launchpad.net/juju/+bug/1951182).

## Release Notes
- Added k8s charm bundle
- Added parallel integration tests
- Added pipelines that rebuild k8s charm bundle on a daily cron job. 

## Testing 
- Currently, there's a small number of semi-relevant integration tests copied over from the pgbouncer-k8s repo. More focused tests coming in a later PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# postgresql-k8s-bundle
\ No newline at end of file
+# postgresql-bundle
```
Comment (marceloneppel) [inline README.md]: I guess the right name is the `postgresql-k8s-bundle`.

**Review by marceloneppel — APPROVED** [review]
LGTM!

**Review by marceloneppel — APPROVED** [review]
LGTM!

**Review by marceloneppel — APPROVED** [review]

**Review by marceloneppel — APPROVED** [review]

**Review by zmraul — APPROVED** [review]
This looks great :)

**Pull request merged**